### PR TITLE
doc: update hvlog kernel parameter descriptions

### PR DESCRIPTION
**Pull request by ZideChen0**
Signed-off-by: Zide Chen <zide.chen@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/user-guides/kernel-parameters.rst [hunk 1]
```diff
@@ -201,11 +201,27 @@ relevant for configuring or debugging ACRN-based systems.
...
+       If hypervisor relocation is disabled, need to check against
+       CONFIG_HV_RAM_START and CONFIG_HV_RAM_SIZE to ensure that the buffer
+       is not overlap with hypervisor's reserved space, provided that in SOS,
+       GPA and HPA is 1:1 mapping.
+
+       If hypervisor relocation is enabled, must reserve the memory below
+       256MB, since hypervisor could be relocated anywhere between 256MB and
+       4GB.
+       
+       Also it is recommended to enable ASLR on SOS. This is to ensure
```
Comment (dbkinder) [inline doc/user-guides/kernel-parameters.rst:218]: change ``Also it is recommended to enable`` to ``You should enable``
change ``This is to ensure that`` to ``This ensures``

Comment (ZideChen0) [reply doc/user-guides/kernel-parameters.rst]: @dbkinder Thank you very much for the fixes! I have updated the PR.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**